cfy_manager/components/__init__.py

```diff
@@ -32,7 +32,7 @@
 from .sanity import SanityComponent  # NOQA
 from .stage import StageComponent  # NOQA
 from .usage_collector import UsageCollectorComponent  # NOQA
-from .cluster import ClusterComponent  # NOQA
+from cfy_manager.components.cluster.cluster import ClusterComponent  # NOQA
```

Comment:
Why?

Reply:
I changed the cluster to be in a directory of his own and didn't notice :)
fixed